libtextsecure/outgoing_message.js

```diff
@@ -126,8 +148,8 @@ OutgoingMessage.prototype = {
         )
         .then(devicesPubKeys => {
           if (devicesPubKeys.length === 0) {
```

Comment:
does `getAllDevicePubKeysForPrimaryPubKey` return `number` if it doesn't have any device mappings?

Reply:
returns in all cases an array even with no secondary device:

```js
  async function getAllDevicePubKeysForPrimaryPubKey(primaryDevicePubKey) {
    await saveAllPairingAuthorisationsFor(primaryDevicePubKey);
    const secondaryPubKeys =
      (await getSecondaryDevicesFor(primaryDevicePubKey)) || [];
    return secondaryPubKeys.concat(primaryDevicePubKey);
  }
```